PR for debug help

weili1988 opened the pull request:
I combined and adjusted the code to Person.java and DurablePersonNGTest.java. I run DurablePersonNGTest.java
Got error like: 
java.lang.NullPointerException  at org.apache.mnemonic.DurablePersonNGTest.testCheckPeople(DurablePersonNGTest.java:152)

Can you give some hint? This is like the last time error. But I think I fixed the patch already.

AI-Memory commented on mnemonic-core/src/test/java/org/apache/mnemonic/DurablePersonNGTest.java:
```diff
@@ -63,6 +63,16 @@ public boolean reclaim(Long mres, Long sz) {
     long val;
+    MemBufferHolder<?> mbh;
+    int size = rand.nextInt(1024 * 1024) + 1024 * 1024;
+    mbh = act.createBuffer(size);
```
Please createBuffer() along with person.setNamebuffer() otherwise multiple person would hold the same buffer.

weilicsu committed:
MNEMONIC-97 debug for null pointer

MNEMONIC-97 debug for null pointer

MNEMONIC-97: Debug for null pointer

MNEMONIC-97: Debug request

Mnemonic-97: debug

weili1988 commented:
I changed accordingly but still I got the same error. Can you pull it again and take a look? Thanks you so much
